app/src/main/java/co/nimblehq/compose/ecommerce/ui/bottomnavigationbar/NavigationItem.kt

```diff
@@ -0,0 +1,39 @@
+package co.nimblehq.compose.ecommerce.ui.bottomnavigationbar
+
+import androidx.annotation.DrawableRes
+import co.nimblehq.compose.ecommerce.R
+
+sealed class NavigationItem(
+    val route: String,
+    @DrawableRes val iconResId: Int,
+    @DrawableRes val selectedIconResId: Int,
+    val title: String
+) {
+    object Home : NavigationItem(
+        route = "home",
+        iconResId = R.drawable.ic_tab_home,
+        selectedIconResId = R.drawable.ic_tab_home_selected,
+        title = "Home"
```

Comment:
Is there a way to use strings from xml config instead of hard-coding here?

Reply:
@luongvo Actually, the title should be used from the data class's property `val title: String`, I did refactor on next upcoming PR 😄